client/app/components/onboardingGuide/GuideStatusPage.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideStatusPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  const onboarding_data = props.onboarding_data;
+  // TODO: interpolating translation strings needs more thinking
+  const title = props.nextStep ? props.t('title').replace(/\%\{(\w+)\}/g, props.name) : props.t('title_done');
+
+
+  const todo_desc_partial =
+    <div>
+      <p className={ scss.description } >
+        { props.t('description_p1') }
+      </p>
+      <p className={ scss.description } >
+        { props.t('description_p2') }
+      </p>
+    </div>
+
+  const done_desc_partial =
+    <p className={ scss.description }
+      dangerouslySetInnerHTML={{ __html: props.t('congratulation') }} ></p>
+
+  const description = props.nextStep ? todo_desc_partial : done_desc_partial;
+  return (
+    <div className="container">
+      <h2 className={ scss.title } >{ title }</h2>
+
+      { description }
+
+      <hr className={ scss.sectionSeparator } />
+
+      <ul className={ scss.stepList } >
+        <li className={ scss.stepListItemDone }>
+          <span className={ scss.stepListLink } >
+            <span className={ scss.stepListCheckbox }></span>
+            Create marketplace
+          </span>
+        </li>
+        { Object.keys(onboarding_data).map(function (key) {
+            let stepListItem = onboarding_data[key].status ? scss.stepListItemDone : scss.stepListItem;
```

Comment:
I'll change this. 
- 'var' shouldn't be used. 
- 'const' should be a default and 
- 'let' is for reassignment situations like counters in loops and stuff.
(There was a 'for' loop at some point - but I don't remember was there any real use for 'let' even then. Bad code anyways.)